scripts/webpack/webpack.config.perf.ts

```diff
@@ -43,15 +39,25 @@ const webpackConfig: webpack.Configuration = {
     ],
   },
   plugins: [
-    new ForkTsCheckerWebpackPlugin({ tsconfig: paths.perf('tsconfig.json') }),
-    new (CopyWebpackPlugin as any)([
-      {
-        from: paths.perfSrc('index.html'),
-        to: paths.perfDist(),
+    new EnvironmentPlugin(getDefaultEnvironmentVars()),
```

Comment:
This should have `process.env.PERF = true`

Reply:
yup it does.